x/lightclient/keeper/msg_server_update_verification_flags.go

```diff
@@ -0,0 +1,33 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authoritytypes "github.com/zeta-chain/zetacore/x/authority/types"
+	"github.com/zeta-chain/zetacore/x/lightclient/types"
+)
+
+// UpdateVerificationFlags updates the light client verification flags.
+// This disables/enables blocks verification of the light client for the specified chain.
+// Emergency group can disable flags while operation group can enable/disable
```

Comment:
Is this correct? Maybe we should update this to be more accurate 
Also, mention the condition where one is being enabled and the other is being disabled ( which would still require the Operational group)

Reply:
Updated the comment 785e788c